### ISSUE TITLE
AbstractForm.IFrameTemplate iframe should have title attribute for accessibility

### ISSUE THREAD

**Issue by dvlpr-o** [gwtbootstrap3/gwtbootstrap3#491]
AbstractForm creates a target `iframe` without a `title` attribute. Accessibility guidelines indicate that `frame` and `iframe` elements should contain a `title` attribute to describe the contents of each frame. This provides a label for the frame so users can determine which frame to enter and explore in detail.

[WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/H64.html)
[Quick Reference Guide to Section 508 Requirements and Standards](https://section508.gov/content/quick-reference-guide#1194.22i)

**Comment by crevete** [comment]
This iframe is created just for technical use (for example : pick the server response once submitted). So technically, it's not mandatory. Just to be clear, what you want is something like this ?
```
<iframe src="javascript:;" name="something" tabindex="-1" title="Form submit helper frame" 
style="position:absolute;width:0;height:0;border:0">
```
An iframe with a "static" title attribute ?

**Comment by dvlpr-o** [comment]
Yes. A static title attribute would do the trick. I'm assuming that browsers won't let you target a hidden `iframe`. Is that why the `iframe` template has this inline styling instead of an HTML5 `hidden` attribute and an inline style rule `display: none:` to hide the `iframe` from all users (including assistive technology)?
